plugins/wasi_nn/whispercpp.cpp

```diff
@@ -196,6 +287,31 @@ Expect<ErrNo> load(WasiNNEnvironment &Env, Span<const Span<uint8_t>> Builders,
         "given parameters...Done"sv);
   }
 
+  // Check the language.
+  if (GraphRef.SpokenLanguage != "auto"sv &&
+      whisper_lang_id(GraphRef.SpokenLanguage.c_str()) == -1) {
+    spdlog::error("[WASI-NN] Whisper backend: Error: unknown language {}."sv,
+                  GraphRef.SpokenLanguage);
+    Env.NNGraph.pop_back();
+    return ErrNo::InvalidArgument;
+  }
+
+  // Check the translate option.
+  if (!whisper_is_multilingual(GraphRef.WhisperCtx)) {
+    if (GraphRef.SpokenLanguage != "en"sv || GraphRef.Translate) {
```

Comment:
I didn't get it. Why do we disable the translate option if the translate is true?